src/compiler/pre-transform/codemods/legacy-story.test.ts

```diff
@@ -0,0 +1,195 @@
+import type { Component } from 'svelte/compiler';
+import { print } from 'svelte-ast-print';
+import { describe, it } from 'vitest';
+
+import { transformLegacyStory } from './legacy-story';
+
+import { parseAndExtractSvelteNode } from '#tests/extractor';
+
+describe(transformLegacyStory.name, () => {
+  it("it moves 'autodocs' prop to 'tags' correctly", async ({ expect }) => {
+    const code = `
+      <script context="module">
+        import { Story } from "@storybook/addon-svelte-csf";
+      </script>
+
+      <Story name="Default" autodocs />
+    `;
+    const node = await parseAndExtractSvelteNode<Component>(code, 'Component');
+
+    expect(print(transformLegacyStory({ node }))).toMatchInlineSnapshot(
+      `"<Story name="Default" tags={["autodocs"]} />"`
+    );
+  });
+
+  it("moving 'autodocs' prop doesn't break with existing 'tags' prop", async ({ expect }) => {
```

Comment:
can we add two more tests to support what we discussed in https://github.com/storybookjs/addon-svelte-csf/pull/207#discussion_r1739989511:

1. `tags="singular"` is converted to `tags=["singular"]`
2. `autodocs tags="singular"` is converted to `tags=["autodocs", "singular"]`

Reply:
I will, I need to create a codemod for this one.
I'll do it tomorrow, because it is a wrap for me today.

Reply:
09e65099b1fc810b3bc8bb5981133cfa41ac7fb6 and 3c361d487720e0bc213cf046bd218015cdbc7576 should cover it.